README.md

```diff
@@ -1,4 +1,10 @@
 # testRepository
 Repositorio de prueba
 Archivo Readme
-Lorem Ipsu
\ No newline at end of file
+Lorem Ipsu
+
+
+
+## CAMBIOS EDU
+bla bla bla
```

Comment:
Qué significa este "bla bla bla"? xD

Reply:
hahahaha this is an example of file with changes